x-pack/legacy/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -130,6 +167,11 @@ export function dataFrameAnalyticsRoutes({ xpackMainPlugin, router }: RouteIniti
     })
   );
 
+  /**
+   * @api {post} /api/ml/data_frame/_evaluate Gets stats for requested analytics job
+   * @apiName EvaluateDataFrameAnalytics
+   * @apiGroup DataFrameAnalytics
```

Comment:
How do we show the schema of what should be in the body of this request? As in 
```
export const dataAnalyticsEvaluateSchema = {
  index: schema.string(),
  query: schema.maybe(schema.any()),
  evaluation: schema.maybe(
    schema.object({
      regression: schema.maybe(schema.any()),
      classification: schema.maybe(schema.any()),
    })
  ),
};
```

Reply:
ideally would be should create a custom parser which imports the schema definition and produces the same output as `@apiParam`

Reply:
As discussed, I think it's worth looking at create a custom parser to handle the schema definition. But one for a follow-up PR!

Reply:
> Is it possible to control the order in which the APIs are listed for a group? For example, it would be better I think if the order matched the order in `data_frame_analytics.ts`, where the more widely used calls appear at the top of the file. The order I see here is quite different:
> 
> data_frame_analytics.ts
> ![image](https://user-images.githubusercontent.com/7405507/73549324-4682f180-443a-11ea-9a12-c91540262d05.png)

I did try to provide a custom order in `apidoc.json` but it seems it doesn't work with groups and names. There are multiple issues opened in apidoc repo regarding the ordering issues, hope it'll be resolved soon.